libs/sdk-backend-tiger/src/backend/user/index.ts

```diff
@@ -11,6 +12,9 @@ export class TigerUserService implements IUserService {
     }
 
     public async getUser(): Promise<IUser> {
-        throw new NotSupported("not supported");
+        return this.authCall(async (sdk) => {
+            const profile = await sdk.axios.get<IUserProfile>("/api/profile");
```

Comment:
Please add a TODO to replace this once the generated client supports this (we ideally do not want any URLs in the sdk-backend-* code).